### DISCUSSION TITLE
Cannot update panels with ESPhome

### DISCUSSION THREAD

**Discussion by rploeg** [Blackymas/NSPanel_HA_Blueprint#1816] [General]
Since two weeks I have problems to update the NSpanels with the new ESPHome version. On every device I get this error:

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/49752333/2247d33b-fd16-42a7-bf70-a9b3e9408c1b)

I have all my keys in my secrets in ESPhome and those are not changed. Any idea how to resolve this? 

I am running latest HA.

**Comment by edwardtfn** [comment]
Could you please share your panel's yaml? Please remove any sensitive info before sharing.

**Comment by edwardtfn** [comment]
Let's continue this on the issues area:

- #1820
